slash/css/src/Table/Pager.scss

```
@@ -13,12 +13,13 @@
   }
 
   &__item {
-    border-radius: common.px-to-rem(25);
     font-size: 0.9375rem;
 
-    &--active {
+    &-link[aria-current] {
```

ici je forcerais sur 
```suggestion
    &-link[aria-current="page"] {
```
afin d'être plus précis